WordPress/WordPressUITests/Tests/SitePageTests.swift

```diff
@@ -0,0 +1,51 @@
+import XCTest
+
+class SitePageTests: XCTestCase {
+    private var sitePageEditorScreen: SitePageScreen!
+    override func setUp() {
```

Comment:
Again, most of the style guides agree on the fact that methods/functions declarations should be preceded by a blank line:

```suggestion
    private var sitePageEditorScreen: SitePageScreen!
    
    override func setUp() {
```